pkgs/development/interpreters/python/cpython/default.nix

```diff
@@ -344,6 +344,8 @@ in with passthru; stdenv.mkDerivation {
     "ac_cv_computed_gotos=yes"
     "ac_cv_file__dev_ptmx=yes"
     "ac_cv_file__dev_ptc=yes"
+  ] ++ optionals (stdenv.hostPlatform != stdenv.buildPlatform && pythonAtLeast "3.11") [
+    "--with-build-python=${pythonForBuildInterpreter}"
```

Comment:
Meh, python needs python when cross-compiling? :(

Reply:
This flag is new in python 3.11, but the requirement seems to exist long ago.